### DISCUSSION TITLE
What happened to `maturin new`?

### DISCUSSION THREAD

**Discussion by marcpabst** [PyO3/maturin#2082] [Q&A] [answered]
### Bug Description

Commands `maturin new` and `maturin init` don't work but are mentioned in the docs/tutorial:

```bash
$ maturin new
error: unrecognized subcommand 'new'
```

### Your maturin version (`maturin --version`)

1.1.5.1

### Your Python version (`python -V`)

3.11

### Your pip version (`pip -V`)

24.0

### What bindings you're using

None

### Does `cargo build` work?

- [ ] Yes, it works

### If on windows, have you checked that you aren't accidentally using unix path (those with the forward slash `/`)?

- [ ] Yes

### Steps to Reproduce

see above

**Comment by messense** [comment, marked as answer]
If you are compiling maturin from source, you need to enable the `scaffolding` feature to use `new`/`init` command.

**Reply by marcpabst** [reply]
Ah right! I never realised that I'm still on a custom-built version of maturin! Thanks!

**Comment by Strus** [comment]
Maturin 1.7.1 installed via pip do not have these commands available. Is there a way to enable them when using pip?

**Reply by hwr9912** [reply]
Maturin installed from conda also do not have those commands either, solved by change the channel to conda-forge
